umh.docs.umh.app/content/en/docs/features/postgresql_and_grafana_alerts.md

```diff
@@ -0,0 +1,245 @@
++++
+title = "Alerts in Grafana using PostgreSQL"
+menuTitle = "Alerts in Grafana using PostgreSQL"
+description = ""
+weight = 50
+draft = false
++++
+
+The United Manufacturing Hub utilizes a TimeScaleDB database which is based on
+PostgreSQL. Therefore, you can use the PostgreSQL plugin in Grafana to implement 
+alerts. This step-by-step tutorial will guide you through the implementation of 
+the PostgreSQL plugin, the creation of an alert rule, and its connection to the
+desired contacts and services.
+
+
+## Why should I use alerts?
```

Comment:
Can you adjust it so that the headings are similar to the other features? in this case it would be "When should I use it?"